{"kind":"symbol","metadata":{"externalID":"s:7Amplify12StorageErrorO7unknownyACSS_s0C0_pSgtcACmF","title":"StorageError.unknown(_:_:)","role":"symbol","modules":[{"name":"Amplify"}],"roleHeading":"Case","symbolKind":"case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:7Amplify16ErrorDescriptiona","text":"ErrorDescription","kind":"typeIdentifier"},{"kind":"text","text":", "},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"kind":"text","text":"?)"}]},"schemaVersion":{"major":0,"minor":3,"patch":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"ErrorDescription","preciseIdentifier":"s:7Amplify16ErrorDescriptiona","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/ErrorDescription"},{"kind":"text","text":", "},{"text":"_","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"kind":"text","text":"? = nil)"}],"platforms":["iOS"]}]}],"abstract":[{"type":"text","text":"Surfaced when a storage operation encounters an general unexpected error."}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/StorageError\/unknown(_:_:)"},"variants":[{"paths":["\/documentation\/amplify\/storageerror\/unknown(_:_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/amplify-swift.Amplify\/documentation\/Amplify","doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/StorageError"]]},"sections":[],"references":{"x-source-tag://StorageCategoryPlugin":{"title":"StorageCategoryPlugin","titleInlineContent":[{"type":"text","text":"StorageCategoryPlugin"}],"type":"link","url":"x-source-tag:\/\/StorageCategoryPlugin","identifier":"x-source-tag:\/\/StorageCategoryPlugin"},"doc://amplify-swift.Amplify/documentation/Amplify/ErrorDescription":{"url":"\/documentation\/amplify\/errordescription","title":"ErrorDescription","type":"topic","abstract":[{"type":"text","text":"See: "},{"type":"reference","identifier":"x-source-tag:\/\/AmplifyError.errorDescription","isActive":true}],"identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/ErrorDescription","fragments":[{"kind":"keyword","text":"typealias"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ErrorDescription"}],"role":"symbol","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"ErrorDescription"}]},"x-source-tag://AmplifyError.errorDescription":{"title":"AmplifyError.errorDescription","titleInlineContent":[{"type":"text","text":"AmplifyError.errorDescription"}],"type":"link","url":"x-source-tag:\/\/AmplifyError.errorDescription","identifier":"x-source-tag:\/\/AmplifyError.errorDescription"},"doc://amplify-swift.Amplify/documentation/Amplify/StorageError":{"abstract":[{"type":"text","text":"Errors thrown by implementations of the"},{"type":"text","text":" "},{"type":"reference","identifier":"x-source-tag:\/\/StorageCategoryPlugin","isActive":true},{"type":"text","text":" protocol."}],"kind":"symbol","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/StorageError","title":"StorageError","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"StorageError"}],"type":"topic","url":"\/documentation\/amplify\/storageerror","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"StorageError"}]},"doc://amplify-swift.Amplify/documentation/Amplify":{"url":"\/documentation\/amplify","type":"topic","title":"Amplify","role":"collection","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify","kind":"symbol","abstract":[]},"doc://amplify-swift.Amplify/documentation/Amplify/StorageError/unknown(_:_:)":{"kind":"symbol","title":"StorageError.unknown(_:_:)","abstract":[{"type":"text","text":"Surfaced when a storage operation encounters an general unexpected error."}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"unknown","kind":"identifier"},{"kind":"text","text":"("},{"text":"ErrorDescription","preciseIdentifier":"s:7Amplify16ErrorDescriptiona","kind":"typeIdentifier"},{"kind":"text","text":", "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"text":"?)","kind":"text"}],"identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/StorageError\/unknown(_:_:)","type":"topic","url":"\/documentation\/amplify\/storageerror\/unknown(_:_:)","role":"symbol"}}}